wolox-books/src/app/screens/register/register.component.ts

```diff
@@ -30,8 +31,14 @@ export class RegisterComponent {
   registerUser() {
     this.userService.createUser({ user: this.registerForm.value })
     .subscribe(
-      resp => { console.log('success') },
+      resp => {
+         console.log('success');
+         this.goTo('login');
```

Comment:
Use the `router.navigate I think there's no need to wrap just that redirect in another class method